azurerm/internal/services/containers/tests/container_registry_resource_test.go

```diff
@@ -781,3 +825,70 @@ resource "azurerm_container_registry" "test" {
 }
 `, data.RandomInteger, data.Locations.Primary)
 }
+
+func testAccAzureRMContainerRegistry_policies(data acceptance.TestData, days int) string {
+	return fmt.Sprintf(`
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-aks-%d"
+  location = "%s"
+}
+
+resource "azurerm_container_registry" "test" {
+  name                = "testacccr%d"
+  resource_group_name = azurerm_resource_group.test.name
+  location            = azurerm_resource_group.test.location
+  admin_enabled       = false
+  sku                 = "Premium"
+
+  policies {
+    retention_policy {
+      days   = %d
+      status = "enabled"
+    }
+
+    trust_policy {
+      status = "enabled"
+    }
+  }
+
+  tags = {
+    environment = "production"
```

Comment:
could we nsure capitals work here
```suggestion
    Environment = "Testing"
```